CHANGES.md

```diff
@@ -28,6 +28,19 @@ OpenSSL 3.3
 
 ### Changes between 3.2 and 3.3 [xx XXX xxxx]
 
+ * Unknown entries in TLS SignatureAlgorithms, ClientSignatureAlgorithms
+   config options and the respective calls to SSL[_CTX]_set1_sigalgs() and
+   SSL[_CTX]_set1_client_sigalgs() that start with `?` character are
+   ignored and the configuration will still be used.
```

Comment:
Is that ignored if it's unknown, but used if it's known?

Reply:
I meant that the whole configuration will be still used even if the list items marked with `?` are unknown.